app/(gcforms)/[locale]/(support)/contact/components/client/ContactForm.tsx

```diff
@@ -69,6 +69,11 @@ export const ContactForm = () => {
         </p>
       </Alert.Warning>
       <form id="contactus" action={formAction} noValidate>
+        {state.error && (
+          <Alert.Danger focussable={true} title={t("error")} className="mb-2 mt-2">
+            <p>{t(state.error)}</p>
```

Comment:
I didn't notice until I was looking at a similar PR. Currently this will print out the untranslated error from the server action because a translation key isn't returned.
![Screenshot 2024-05-24 at 10 27 58 AM](https://github.com/cds-snc/platform-forms-client/assets/107579368/d6edb6fd-409d-4f05-b9d9-7d2c59e30e2b)


We could update the server action to instead return a translation key? Or just hard code a translation key in the error since only one error exists. In either case, we'd probably want some new content for this since I don't think we have an error message for this.

I could approve this PR and the error content could be updated in another PR?